docs/application/dotnet/get-started/wearable/first-app.md

```diff
@@ -1,183 +1,124 @@
 
-    This step shows how you can deploy and run the application on the
-    emulator or a real target device.
+    This step shows how you can deploy and run the application on the emulator or a real target device.
 
-5. [Enhancing Your Application](#enhancing-your-application).
+5. [Enhance Your Application](#enhance-your-application).
 
-    This step shows how you can enhance your application by creating a
-    UI and making small alterations to it to improve the usability of the application.
+    This step shows how you can enhance your application by creating a UI and making small alterations to it to improve the usability of the application.
 
-## Creating a Project
+## Create a Project
 
-The following example shows you how to create and configure a basic
-Tizen .NET application project in Visual Studio. An application project
-contains all the files that make up an application.
+The following example shows you how to create and configure a basic Tizen .NET application project in Visual Studio. An application project contains all the files that make up an application.
 
-The following figure illustrates the output of application. The
-application screen displays a message, **Welcome to Xamarin Forms!** and there is no user interaction.
+The following figure illustrates the output of application. 
 
-**Figure: Application running on the wearable emulator**
+![Application running on the wearable emulator](media/cs_first_building_emulator_wearable.png)
 
-![Application running on the wearable
-emulator](media/cs_first_building_emulator_wearable.png)
+The application screen displays a message, **Welcome to Xamarin Forms!** and there is no user interaction.  
 
 To create a new Tizen .NET project:
 
-1.  Launch Visual Studio.
-2. In the Visual Studio menu, select **File &gt; New &gt; Project**.
+1. Launch Visual Studio 2019.
+
+2. In the Visual Studio menu, select **File \> New \> Project**.
 
     ![Create a new project](media/cs_first_creating_menupath.png)
 
     A New Project window appears.
 
-3. Select **Installed &gt; Visual C\# &gt; Tizen** and select
-    **Tizen App (Xamarin.Forms)** template.
-
-    Define the project properties and click **OK**. You can enter the **Name**, **Location**, and **Solution name**.
+3. Select **C\#** in languages and **Tizen** in platforms, select **Blank App (Xamarin.Forms)** template, and then click **Next**.
 
     ![Select a template](media/cs_first_creating_new_project.png)
 
+    Configure the project properties and click **Create**. You can enter the **Project name**, **Location**, **Solution**, and **Solution name**.
+
+    ![Select a template](media/cs_first_creating_configure_project.png)
+
     The **Tizen Project Wizard** pop-up window appears.
 
-4. Select the profile, **Wearable (preview)**. Click **OK**.
+4. Select the profile, **Wearable (preview)** and click **OK**.
 
     ![Tizen Project Wizard](media/cs_first_creating_project_wizard.png)
```

Comment:
Minor issue.

Compared to the update in mobile and tv,
the red box is missing.

Update in mobile : 
![image](https://user-images.githubusercontent.com/9958374/74010619-44380e80-49c9-11ea-9baf-24c30e82d958.png)

Update in tv : 
![image](https://user-images.githubusercontent.com/9958374/74010654-57e37500-49c9-11ea-90cf-8a198659dca4.png)

Update in wearable : 
![image](https://user-images.githubusercontent.com/9958374/74010678-63cf3700-49c9-11ea-81c8-563fc1414f2b.png)

Reply:
It's my intention. Wearable (preview) has a box alreay.